src/extremeweatherbench/events/atmospheric_river.py

```diff
@@ -0,0 +1,255 @@
+import numpy as np
+import regionmask
+import scores.categorical as cat
+import xarray as xr
+from scipy import ndimage
+from skimage import filters
+
+from extremeweatherbench import calc
+
+
+def atmospheric_river_mask(
+    data: xr.Dataset,
+    laplacian_threshold: float = 2.5,
+    ivt_threshold: float = 400,
+    dilation_radius: int = 8,
+    min_size_gridpoints: int = 500,
+) -> xr.DataArray:
+    """Calculate the atmospheric river mask using thresholds for IVT and its Laplacian.
+
+    The incoming dataset must contain the variables integrated_vapor_transport
+    and integrated_vapor_transport_laplacian. The current implementation uses
+    standard grid spacing of 0.25 degrees (same as ERA5); users must convert their
+    data to this grid spacing before using this function as of v1.0.0.
+
+    Parameter defaults for the mask are based on Newell et al. 1992, Mo 2024,
+    TempestExtremes v2.1 criteria (Ullrich et al. 2021), and visual inspection of
+    ERA5 outputs.
+
+    Args:
+        data: the input dataset containing integrated_vapor_transport as a variable and
+        integrated_vapor_transport_laplacian as a variable.
+        laplacian_threshold: the threshold for the Laplacian in kg/m^2/s^2
+        ivt_threshold: the threshold for the IVT in kg/m/s
+        dilation_radius: the radius for the dilation of the Laplacian in gridpoints
+        min_size_gridpoints: the minimum size of the atmospheric river in gridpoints
+    Returns:
+        The atmospheric river mask
+    """
+    # Get all coordinates except level for the intersection DataArray
+    coords_dict = {dim: data.coords[dim] for dim in data.dims if dim != "level"}
+    # Create boolean masks for each condition
+    has_high_laplacian = (
+        np.abs(data["integrated_vapor_transport_laplacian"]) >= laplacian_threshold
+    )
+    has_high_ivt = data["integrated_vapor_transport"] >= ivt_threshold
+    # For the Laplacian condition, we want to check if there's a value >= 2.5 within 8
```

Comment:
Same nit as before - try to include a newline between commented blocks of code, helps maintain readability.